src/NuGet.Services.Metadata.Catalog.Monitoring/ValidationCollector.cs

```diff
@@ -43,7 +43,7 @@ protected override async Task ProcessSortedBatchAsync(CollectorHttpClient client
 
             _logger.LogInformation("Processing catalog entries for {PackageId} {PackageVersion}.", packageId, packageVersion);
 
-            var catalogEntries = sortedBatch.Value.Select(c => new CatalogIndexEntry(c));
+            var catalogEntries = sortedBatch.Value.Select(token => CatalogIndexEntry.Create(token));
```

Comment:
>token => CatalogIndexEntry.Create(token) [](start = 58, length = 40)

You can save on delegate creation and pass `CatalogIndexEntry.Create` directly as a delegate.